app/workers/work-reaper.ts

```diff
@@ -30,19 +31,25 @@ export default class WorkReaper implements Worker {
         db, notUpdatedForMinutes, jobStatus,
       );
       if (workItemIds.length) {
-        const numItemsDeleted = await deleteWorkItemsById(db, workItemIds);
-        this.logger.info(`Work reaper removed ${numItemsDeleted} work items`);
+        const chunkedWorkItemIds = _.chunk(workItemIds, batchSize);
+        for (const workItemIdsChunk of chunkedWorkItemIds) {
+          const numItemsDeleted = await deleteWorkItemsById(db, workItemIdsChunk);
+          this.logger.debug(`Work reaper removed ${numItemsDeleted} work items`);
```

Comment:
I think logging at info level was appropriate so we could see what was being cleaned up. Anything at debug level will not be logged in a deployed environment normally.